### ISSUE TITLE
Ubuntu 20.04 is here

### ISSUE THREAD

**Issue by Neustradamus** [serghey-rodin/vesta#1999]
Ubuntu 20.04 is here
- https://ubuntu.com/

**Comment by khards** [comment]
Is there a Vesta roadmap?

**Comment by ScIT-Raphael** [comment]
@khards probaly the same roadmap as for the zero day exploits that currently are out in the wild: https://github.com/serghey-rodin/vesta/issues/1984

**Comment by khards** [comment]
I am migrating to a new server, and have the choice of using discontinued 18.04 that won't update or 20.04 which I just got my application working with. The problem I have is that no panel (except possibly ispconfig) is compatible with 20.04.
I think I'll end up configuring it myself as it's just 1 website. Just a PITA manually configuring the domain and DB user, rather than installing vesta and letting that do the heavy lifting like SSL certs.
I'm sure it'll be along soon enough, but I can't really delay any longer as I need to complete my server move during LOCKDOWN-20

**Comment by ScIT-Raphael** [comment]
Ubuntu 20.04 is quiet new, also our fork isnt ready due to the missing external repository support. But this will follow within one or two weeks.

If you only have one website and want to use Ubuntu 20.04 now, the only way would be to do a manual installation.

**Comment by Neustradamus** [comment]
@ScIT-Raphael: Debian 10 and RHEL 8.0 + CentOS 8.0 soon one year ;)
- https://github.com/serghey-rodin/vesta/issues/1956
- https://github.com/serghey-rodin/vesta/issues/1957
- https://github.com/serghey-rodin/vesta/issues/2000

**Comment by skironDotNet** [comment]
@Neustradamus I guess it's time to move to Ubuntu like the rest of the world ;)

**Comment by aligoren** [comment]
I think we will wait more :/ I wish to install 18.x version of ubuntu.

**Comment by AiratHalitov** [comment]
When will be added support of Ubuntu 20.04? In this year?

**Comment by ScIT-Raphael** [comment]
> @khards probaly the same roadmap as for the zero day exploits that currently are out in the wild: #1984

still the same release date.

**Comment by lianglee** [comment]
I guess you guys are working on new version for Ubuntu 20.04 ?

**Comment by anton-reutov** [comment]
yes, we are working under new installers for all actual versions
